docs/manual/developer/10_stabilization_phase.md

```diff
@@ -0,0 +1,22 @@
+# Stabilization phase
+
+## Definition
+A period of time before release, when the project is extensively tested and the only accepted contributions are fixes to issues discovered during stabilization testing. New content, features, tests, or improvements are not accepted. Fixes for known issues that haven't been discovered during the stabilization process are also not accepted.
+
+To not interfere with project liveliness, the stabilization happens in a separate branch with *stabilization* prefix, e.g. stabilization-v0.1.70. Master/main branch contributions are not affected.
+
+During this period, project is tested, contributors are informed about upcoming release, everything is prepared for release (release notes, contributors list, milestones), and issue fixes are verified.
+
+## Duration
+Current duration is set to **2 weeks**. This duration is approximate. Complications such as newly discovered major issues during this period might slightly prolong it.
+
+The duration is tied to release cadence. The more time between release, the more time for stabilization is needed. Thus, the current 2 weeks stabilization is for release every 2 months. However, stabilization period does not affect release cadence. Next release date will be the same as there would not be any prolongation.
```

Comment:
For now I agree with this sentence `The more time between release, the more time for stabilization is needed.` but it should be reviewed in the future because along the time we can also optimize the stabilization phase and therefore the cadence should have minimal impact on time demanded for stabilization phase.

Reply:
Yes, with optimized stabilization phase it will take less time. But even then the highlighted sentence will be true. For example we can get to 1 week long stabilization for release every 2 months, but if we change cadence to 3 months, the stabilization will need to be slightly prolonged as the chance of problems will be higher.